dawn/renderer/components/UpdateBox.js

```diff
@@ -52,25 +53,66 @@ class UpdateBox extends React.Component {
 
   upgradeSoftware() {
     this.setState({ isUploading: true });
+    const update = Editor.pathToName(this.state.updateFilepath);
+    const signat = Editor.pathToName(this.state.signatureFilepath);
```

Comment:
Signature files are deprecated. Last I checked, we decided not to bother with the extra hassle and use update files only.